### PR TITLE
Add grid utility components

### PR DESCRIPTION
**Pull request by hdfedscottquested**
@hdfeddanjackson I had already committed these straight to development branch before realising. So couldn't revert to create a feature branch :(

### CODE REVIEW

File: components/utility/layout/Container.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const Container = props => {
+
+    const {
+        fluid = false,
+        className = '',
+        children
+    } = props
+
+    const containerClass = classNames('g-container', className, {
+        'g-container--fluid': fluid,
```
Comment (hdfeddanjackson) [inline components/utility/layout/Container.js:15]: Do you need the comma at the end of line 15?

Comment (hdfedscottquested) [reply components/utility/layout/Container.js:15]: Nope - good spot

Comment (hdfedscottquested) [reply components/utility/layout/Container.js]: Done

File: src/scss/global/sizing.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+html {
+	box-sizing: border-box;
+}
+*, *:before, *:after {
```
Comment (hdfeddanjackson) [inline src/scss/global/sizing.scss:4]:
```scss
*, 
*:before, 
*:after {
```

Could this perhaps be on a single line as per our coding guidelines?

Comment (hdfedscottquested) [reply src/scss/global/sizing.scss]: Yep - another good spot. Fixed

File: src/scss/mixins/column-mixin.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+			.g-col#{$modifier}-offset-0 {
+				margin-left: 0;
+			}
+
+			@for $i from 1 through $grid-cols {
+
+				.g-col#{$modifier}-#{$i} {
+
+					width: (100 / ($grid-cols / $i) ) * 1%;
```
Comment (hdfeddanjackson) [inline src/scss/mixins/column-mixin.scss:29]: Should we amend this:

```scss
width: (100 / ($grid-cols / $i)) * 1%;
```

File: src/scss/mixins/column-mixin.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+					padding: 1em;
+					float: none;
+				}
+
+				.g-col#{$modifier}-offset-#{$i} {
+					margin-left: (100 / ($grid-cols / $i) ) * 1%;
```
Comment (hdfeddanjackson) [inline src/scss/mixins/column-mixin.scss:35]: Should we amend this:

```scss
width: (100 / ($grid-cols / $i)) * 1%;
```

File: src/scss/mixins/column-mixin.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+			.g-col#{$modifier}-max-offset-0 {
+				margin-left: 0;
+			}
+
+			@for $i from 1 through $grid-cols {
+
+				.g-col#{$modifier}-max-#{$i} {
+
+					width: (100 / ($grid-cols / $i) ) * 1%;
```
Comment (hdfeddanjackson) [inline src/scss/mixins/column-mixin.scss:50]: Should we amend this:

```scss
width: (100 / ($grid-cols / $i)) * 1%;
```

**Comment by hdfeddanjackson** [comment]
Should the name of the PR be updated from 'Development'?

File: src/scss/mixins/column-mixin.scss (hunk 3 above)
Comment (hdfedscottquested) [reply src/scss/mixins/column-mixin.scss:29]: Am I not seeing something here? Looks the same

Comment (hdfeddanjackson) [reply src/scss/mixins/column-mixin.scss:29]: The whitespace between the two closing braces.

Comment (hdfedscottquested) [reply src/scss/mixins/column-mixin.scss:29]: Ah!!!! Gotcha

Comment (hdfedscottquested) [reply src/scss/mixins/column-mixin.scss]: Done

**Comment by hdfedscottquested** [comment]
> Should the name of the PR be updated from 'Development'?

@hdfeddanjackson Yes, done

**Review by hdfeddanjackson — APPROVED** [review]
Approved following changes.

**Pull request merged**